operations/iuf/stages/management_nodes_rollout.md

```diff
@@ -1,7 +1,19 @@
 # management-nodes-rollout
 
-The `management-nodes-rollout` stage performs a controlled rebuild of the management worker nodes in order to reboot them to a new image and configuration. IUF will account for the necessary minimum number of critical software
-instances running on worker nodes to ensure the `management-nodes-rollout` stage operates without impacting software availability.
+The `management-nodes-rollout` stage performs a controlled rebuild of the management NCNs in order to reboot them to a new image and configuration. IUF will account for the necessary minimum number of critical software
+instances running on the nodes to ensure the `management-nodes-rollout` stage operates without impacting software availability.
+
+**`NOTE`** `management-nodes-rollout` currently does not rebuild management NCN storage nodes or `ncn-m001`. These nodes must be rebuilt using non-IUF methods described in the appropriate sections of the CSM documentation.
```

Comment:
i _think_ it does however set up m001 to rebuild when it's next rebooted.  needs verification.

Reply:
@leliasen-hpe can you comment on this?  I do want to make it clear if they have to follow *all* of the manual steps or if IUF did part of the work for the admin.

Reply:
IUF does not do anything related to m001. This means the admin would have to follow manual steps. However, there is just 1 script that will do the full rebuild for a master node. The admin will be responsible for setting the image id in BSS and CFS config before rebuildd

Reply:
Lindsay is going to follow up on this in CASMINST-5906.